modules/messages/EED_Messages.module.php

```diff
@@ -636,7 +636,10 @@ protected static function _verify_registration_notification_send(
         ) {
             $status_change = $request->getRequestParam('txn_reg_status_change', [], 'int', true);
             // make sure appropriate admin params are set for sending messages
-            if (! $status_change['send_notifications']) {
+            if (
+                ! isset($status_change['send_notifications'])
+                || (isset($status_change['send_notifications']) && ! $status_change['send_notifications'])
+            ) {
```

Comment:
also a good change and won't cause issues because the value of request param is checked.